Assets/Scripts/UIManager.cs

```diff
@@ -0,0 +1,139 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+using UnityEngine.UI;
+
+
+public class UIManager : MonoBehaviour
```

Comment:
This class should not be called `UIManager`, UI manager should be a script managing or setting up the UIs. You are working on the player's status UI, which this class should be called something like `UI_PlayerStatusDisplay`.